requirements-test.txt

```diff
@@ -1,7 +1,7 @@
 nose
 coverage
 coveralls
-responses>0.5.0,<0.9.0
+responses>0.5.0,<0.10.3
```

Comment:
use `<0.11.0` instead of `<0.10.3`, learn more about [semver](https://semver.org/)